hack/cluster-sync-split.sh

```diff
@@ -101,5 +101,8 @@ _kubectl apply --kustomize ./deploy/controller-infra/dev-overlay
 # ******************************************************
 # Wait for driver to rollout
 # ******************************************************
+_kubectl_tenant rollout restart ds/kubevirt-csi-node -n $CSI_DRIVER_NAMESPACE
+_kubectl rollout restart deployment/kubevirt-csi-controller -n $TENANT_CLUSTER_NAMESPACE
+
```

Comment:
why is this necessary?

Reply:
The image name is not changed when it te-built. That why kubectl rollout status does nothing, and the pods are still running with the old imsges.

With this change, the pods will use the latest images that was just built a few lines above, letting you fix code and retry the test.

Reply:
oh nice! good addition